src/backend/opencl/Array.cpp

```diff
@@ -247,8 +247,8 @@ Array<T> createNodeArray(const dim4 &dims, Node_ptr node) {
 
             bool isBufferLimit =
                 lock_bytes > getMaxBytes() || lock_buffers > getMaxBuffers();
-
-            bool isNvidia = getActivePlatform() == AFCL_PLATFORM_NVIDIA;
+            bool isNvidia = getActivePlatform() == AFCL_PLATFORM_NVIDIA ||
```

Comment:
is this parameter named incorrectly or something ? or `AFCL_PLATFORM_APPLE` can be any OpenCL device , can't it be ?

Reply:
After some offline discussion, this is considered okay OSX.